dependency-check-suppressions.xml

```diff
@@ -213,4 +213,28 @@
         <gav regex="true">^org\.springframework\.security:spring-security-crypto:.*$</gav>
         <cpe>cpe:/a:pivotal_software:spring_security</cpe>
     </suppress>
+  <suppress>
+    <notes><![CDATA[
```

Comment:
This shouldn't be suppressed. I have put a fix into master which should resolve this issue if you rebase